pkgs/misc/vim-plugins/update.py

```diff
@@ -532,22 +536,35 @@ def update_plugins(input_file: str, outfile: str, cache: Cache) -> Dict:
 
 def main() -> None:
     args = parse_args()
+    if args.add_plugins and not args.commit:
+        raise Exception("The --add argument requires setting the --commit flag.")
     if args.commit:
         nixpkgs_repo = NixpkgsRepo(args.allow_dirty)
     current_plugins = get_current_plugins()
     cache = Cache(current_plugins)
     redirects = {}
 
     redirects = update_plugins(args.input_file, args.outfile, cache)
-
     rewrite_input(args.input_file, redirects)
 
     if args.commit:
-        nixpkgs_repo.commit("Update", [args.outfile])
+        nixpkgs_repo.commit("vimPlugins: Update", [args.outfile])
```

Comment:
As a minor pet-peeve of mine, I wish we had a standard on capitalization after `:`. Lower-case seems more prevalent, so I would prefer that. Again, I don't feel very strongly about this though.

Reply:
Yeah, I was going with the precedent in `vim.section.md` but now that we're in a position to correct this I'll do so because I think nixpkg's `CONTRIBUTING.md` should be the global standard.